mof_array/pmf/process_mass_data.py

```diff
@@ -564,10 +619,11 @@ def plot_binned_array_pmf_data(gases, list_of_arrays, bins, binned_probabilities
             plot_PMF = plt.figure()
             plt.rc('xtick', labelsize=20)
             plt.rc('ytick', labelsize=20)
-            plt.plot(comps_to_plot, pdfs_to_plot, 'ro')
-            plt.title('Array: %s, Gas: %s' % (array, gas))
+            plt.plot(comps_to_plot, pdfs_to_plot, 'ro-')
             if len(array) <= 40:
-                plt.savefig("figures/%s/%s_%s.png" % (timestamp, array, str(gas)))
+                plt.title('Array: %s, Gas: %s' % (array, gas))
+                plt.savefig("saved_array_pmfs_binned_figures/%s/%s_%s.png" % (timestamp, array, str(gas)))
             else:
-                plt.savefig("figures/%s/%s_%s.png" % (timestamp, 'Array #'+array_id_dict[array]+' (See Key)', str(gas)))
+                plt.title('Array: %s, Gas: %s' % (array_id_dict[array], gas))
+                plt.savefig("saved_array_pmfs_binned_figures/%s/%s_%s.png" % (timestamp, 'Array#'+array_id_dict[array]+'_(See Key)', str(gas)))
```

Comment:
use second string interpolation instead of '' + '' + ''; remove str() around gas if unnecessary